hooks/request/use-influencers.tsx

```
@@ -0,0 +1,118 @@
+      invitationsRef,
+      where("userId", "==", influencerID)
+    );
+
+    const invitedInfluencer = await getDocs(influencersQuery);
+
+    return invitedInfluencer.docs.length > 0;
+  };
+
+  useEffect(() => {
+    const unsubscribe = fetchInitialInfluencers();
+    return () => unsubscribe && unsubscribe();
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)